.github/workflows/create-tag-with-release-pr.yml

```
@@ -0,0 +1,84 @@
+name: Create release tag on release PR merge
+on:
+  pull_request:
+    types:
+     - closed
+
+jobs:
+  create-release-tag:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release')
+    env:
+      GH_REF: ${{ github.event.pull_request.head.ref }}
+      GH_TOKEN: ${{ github.token }}
```

`GH_TOKEN` doesn't appear to be used anywhere?

```suggestion
      GH_REF: ${{ github.event.pull_request.head.ref }}
```

---

It's needed for gh cli commands
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-github-cli-in-workflows